src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerUdpBatcher.cs

```diff
@@ -232,7 +232,7 @@ protected virtual void Dispose(bool isDisposing)
             {
             }
 
-            if (isDisposing && !this.isDisposed)
+            if (disposing && !this.isDisposed)
```

Comment:
Probably rename `isDisposed` to `disposed` https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#implement-the-dispose-pattern.